src/theme/default/menu.m.css

```diff
@@ -3,3 +3,6 @@
 	overflow: auto;
 	position: relative;
 }
+
+.divider {
```

Comment:
just noticed you'd not added a comment to the `divider` class, please can you do that?